variables.tf

```diff
@@ -47,6 +48,12 @@ variable "namespace" {
   description = "Namespace, which could be your organization name or abbreviation, e.g. 'eg' or 'cp'"
 }
 
+variable "tenant" {
+  type        = string
+  default     = null
+  description = "Usually omitted, but available for distinguishing resources dedicated to a specific customer"
```

Comment:
```suggestion
  description = "Usually omitted, but available for distinguishing resources dedicated to a specific customer in a multi-tenant environment"
```

Reply:
"in a multi-tenant environment" is tautological and/or redundant depending on your point of view. If you have resources dedicated to a specific customer, it is by definition a multi-tenant environment.

Reply:
```suggestion
  description = "_(Rarely used)_ A customer identifier, indicating who this instance of a resource is for"
```